docs/using/Outside-Call-Format.md

```diff
@@ -73,7 +73,7 @@ We rely on string matching to match outside calls to recommendations.
 
 Consult the [Phenotypes List](/Phenotypes-List) for a complete list of named alleles, phenotypes and activity scores.
 
-_Do not_ prefix allele names with the gene symbol in the second field (e.g. CYP2C9*1/CYP2C9*3). The gene is specified in
+_Do not_ prefix allele names with the gene symbol in the second field (e.g. `CYP2C9*1/CYP2C9*3`). The gene is specified in
```

Comment:
This is one place where I'm actually ok with using monospace in context since it's talking about the literal string appearing in the text file, not describing the actual diplotype itself.